content/en/Concepts/environment.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Environment"
+linkTitle: "Environment"
+draft: false
+weight: 2
+description: >
+  The common frame that powers your infrastructure.
+---
+
+## What is an Environment?
+
+An Environment is the common frame that powers your infrastructure.
```

Comment:
This is too abstract. Checkout the definition which is here: https://github.com/run-x/opta-docs/pull/150